tasks/ssl/Linux.yml

```diff
@@ -0,0 +1,110 @@
+---
+- name: Check if local key exists
+  stat:
+    path: '{{ minio_local_key_path }}'
+  delegate_to: '{{ minio_local_certs | ternary("localhost", inventory_hostname) }}'
+  register: key_file
+  when: minio_local_certs
+
+- name: Check if local cert exists
+  stat:
+    path: '{{ minio_local_cert_path }}'
+  delegate_to: '{{ minio_local_certs | ternary("localhost", inventory_hostname) }}'
+  register: cert_file
+  when: minio_local_certs
+
+- name: Copy key file
+  copy:
+    src: '{{ minio_local_key_path }}'
+    dest: '{{ minio_key_path }}/private.key'
+    owner: '{{ minio_user }}'
+    group: '{{ minio_user }}'
+    mode: 0644
+    remote_src: '{{ not minio_local_certs }}'
+  become: True
+  when: key_file.stat.exists
+
+- name: Copy cert file
+  copy:
+    src: '{{ minio_local_cert_path }}'
+    dest: '{{ minio_cert_path }}/public.crt'
+    owner: '{{ minio_user }}'
+    group: '{{ minio_user }}'
+    mode: 0644
+    remote_src: '{{ not minio_local_certs }}'
+  become: True
+  when: cert_file.stat.exists
+  notify:
+    - Restart Minio
+
+- name: Check if key exists
+  stat:
+    path: '{{ minio_key_path }}/public.crt'
+  register: key_file
+  become: True
+
+- name: Check if cert exists
+  stat:
+    path: '{{ minio_cert_path }}/{{ minio_ssl_cert_file_name }}'
+  register: cert_file
+  become: True
+
+- block:
+    - name: Enable EPEL repository
+      yum:
+        name: 'https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm'
+        state: present
+      register: package_install
+      until: package_install is succeeded
+      when: ansible_os_family == 'RedHat'
+      become: True
+
+    - name: Load variables for OS family
+      include_vars: '{{ ansible_os_family }}.yml'
+
+    - name: Install requirements
+      package:
+        name: '{{ requirements }}'
+        state: present
+      register: package_install
+      until: package_install is succeeded
+      become: True
```

Comment:
Still need here?

Reply:
Become does not needed because we use become in block. Thank you for your comment.
It's fixed.